### PR TITLE
feat(spv-1062) removing new relic

### PR DESCRIPTION
**Pull request by ac4ch**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

**Review by Nazarii-4chain — APPROVED** [review]
LGTM

**Review by dorzepowski — CHANGES_REQUESTED** [review]

File: cmd/main.go
Comment (chris-4chain) [inline cmd/main.go:1]: **General comment**
I can still see a lot of occurrences of the keyword `relic`.
image

**Review by chris-4chain — CHANGES_REQUESTED** [review]

File: engine/interface.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/bitcoin-sv/go-paymail"
 	"github.com/bitcoin-sv/spv-wallet/engine/chain"
-	"github.com/bitcoin-sv/spv-wallet/engine/chain/models"
+	chainmodels "github.com/bitcoin-sv/spv-wallet/engine/chain/models"
```
Comment (dorzepowski) [inline engine/interface.go:9]: why this change?

Comment (dorzepowski) [reply engine/interface.go:9]: ok, now I know. It's fine

**Review by dorzepowski — APPROVED** [review]

File: engine/interface.go (hunk 1 above)
Comment (nonus25) [reply engine/interface.go:9]: np :) good

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**